packages/notifications/src/pushNotifications/types/pushNotifications.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import type { PushNotificationMessage } from '@aws-amplify/react-native';
+export type {
+	PushNotificationMessage,
+	PushNotificationPermissionStatus,
+	PushNotificationPermissions,
+} from '@aws-amplify/react-native';
+
+export type OnTokenReceivedHandler = (token: string) => any;
```

Comment:
Very nit: can we reduce use of `any` if possible?
```suggestion
export type OnTokenReceivedHandler = (token: string) => void;
```

Reply:
Originally, the thought was that this handler doesn't belong to so it could return whatever - but since we don't return it it's probably best to enforce void after all